benchmarks/graph_infra/arkouda.graph

```diff
@@ -45,3 +45,9 @@ graphkeys: Noop ops/s
 files: noop.dat
 graphtitle: Noop Performance
 ylabel: Performance (ops/s)
+
+perfkeys: intersect Average rate =, union Average rate =, xor Average rate=, diff Average rate=
```

Comment:
Could you move this above the noop graph? (The graphs show up in the order they're listed here and I think this one is more interesting/important than the noop)